<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="operand1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="operation"/>
     </item>
     <item>
      <widget class="QLineEdit" name="operand2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="result"/>
     </item>
     <item>
      <widget class="QPushButton" name="PushButton">
       <property name="text">
        <string>Do some magic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyStateWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyStateWidget</class>
   <extends>QWidget</extends>
   <header>MyStateWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
